buildenv/jenkins/openjdk10_ppc64_aix

```diff
@@ -3,7 +3,7 @@
 	Set parameter TARGET(openjdk, system, external, perf, jck etc.).
 	Set UPSTREAM_JOB_NAME(version_build_arch_os, for example: openjdk8_build_x86-64_linux)
 	Set JVM_VERSION(openjdk8, openjdk8-openj9, openjdk9, openjdk8-openj9, etc.*/
-LABEL='aix&&ppc64&&test'
+LABEL='aix&&ppc64&&test||aix'
```

Comment:
Those label may not work at expected. It could be the build node if using this simple "aix". Same as following changes of labels.